test/com/arpnetworking/metrics/portal/TestBeanFactory.java

```diff
@@ -44,6 +51,49 @@
  */
 public final class TestBeanFactory {
 
+    private static final String TEST_EMAIL = "noreply+email-recipient@test.com";
```

Comment:
All these should be randomly generated or at least randomly selected from sets where generation is not possible.